src/org/usfirst/frc2876/Steamworks2017/Robot.java

```diff
@@ -51,6 +51,10 @@
 	// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
 
 	public static Vision vision;
+	
+	public static final boolean IS_TURN_PID_FUNCTIONAL = false;
+	public static final boolean IS_DISTANCE_PID_FUNCTIONAL = false;
+	public static final boolean IS_STRAIGHT_PID_FUNCTIONAL = false;

```

Comment:
I think the students thought they wanted to give fallbacks for when the PIDs are not working, which they weren't as we ended last Tuesday.  Not sure why in Robot.java except it is sort of the main class?  I'd like to create something like a Constants class next year.  The Outliers (5687) have something like that here: https://github.com/frc5687/2017-protobot/blob/master/src/main/java/org/frc5687/steamworks/protobot/Constants.java